axelrod/history.py

```diff
@@ -130,12 +131,12 @@ def flip_plays(self):
     def append(self, play, coplay):
         """Appends a new (play, coplay) pair an updates metadata for
         number of cooperations and defections, and the state distribution."""
+        # casts plays sent as integers into Action objects
```

Comment:
Is this comment correct? I don't see a change that would cause a cast. (same below)

Reply:
whoops! forgot to remove this.